css/styles.css

```diff
@@ -0,0 +1,543 @@
+@import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@700&display=swap');
+
+body {
+  padding: 0;
+  margin: 0;
+  overflow-x: hidden;
+  margin-bottom: 70px;
+  font-family: sans-serif;
+}
+
+header {
+  border-bottom: 1px solid #b9b6b6;
+  box-sizing: border-box;
+}
+
+.nav-flex-container,
+#nav-right-suscribe,
+.inflation-description {
+  font-family: 'Libre Franklin', helvetica, arial, sans-serif;
+}
+
+h1,
+.inflation-desc,
+.more-img-caption,
+.more-img-desc {
+  font-family: 'georgia', "times new roman", times, serif;
+}
+
+#first-div {
+  display: flex;
+  justify-content: center;
+}
+
+.nav-flex-container {
+  display: flex;
+  justify-content: space-around;
+  padding: 2px 15px;
+  align-items: center;
+  position: fixed;
+  width: 100vw;
+  top: 0;
+  opacity: 50;
+  visibility: visible;
+  z-index: 3;
+  box-shadow: rgba(0, 0, 0, 0.08) 0 0 5px 1px;
+  border-bottom: 1px solid #e2e2e2;
+  background-color: white;
+}
+
+#folded-menu-icon {
+  width: 16px;
+  height: 16px;
+  padding-right: 20px;
+}
+
+#search-icon {
+  width: 16px;
+  height: 16px;
+  padding-right: 20px;
+}
```

Comment:
We have to remove repeated stylings and follow the DRY principle. Please remove all the repeated codes.